### DISCUSSION TITLE
make it amd device compatible please 😭😭😭😭😭

### DISCUSSION THREAD

**Discussion by gxfrjhyigy** [w-okada/voice-changer#633] [Ideas]
that would be so cool if it could be compatible

**Comment by Edward-Lucas** [comment]
I agree with you very much.

**Comment by w-okada** [comment]
Is there a problem with the directML version?

**Reply by AQutsu** [reply]
I have a RX6600, downloaded directML, and im using one of the basic moddels that is onnx. Once i start it my character takes a lot of time to start replying and its really laggy, like it takes 5 seconds or more to say "hey" with cuts and everything. I haven't changed any settings, only applied gpu0, wich I expect is the right one as the directML version doesn't have a slider unlike nvidia. Already redownloaded it like 5 times, two of them i didn't even get a reply. I'm going to keep trying to fix it but im really stuck. oh and im using client

**Reply by AQutsu** [reply]
UPDATE: It works
Okay so i cannot make it use the gpu, been through gpu0 to gpu3 but i still see like only 5-10% gpu and still needs a lot of chunk and optimization and 50-70% cpu, so my guess is that its not using the gpu for conversion. Is there something i could bring to you? like logs or something in case its useful for this AWESOME thing?
